src/components/ContactsList/ContactsEmptyList.jsx

```diff
@@ -42,17 +47,23 @@ class ContactsEmptyList extends React.Component {
     this.props.hideModal()
     return this.props.showModal(<ContactCardModal id={contact.id} />)
   }
+
   render() {
-    const { hasConnector } = this.state
-    const host = window.location.host
-    const isToutaticeInstance =
-      /\.mytoutatice\.cloud$/.test(host) ||
-      /\.testcloud\.toutatice\.fr$/.test(host) //@TODO Some contexts don't have the google connector available. A better way to do this would be to query the list of available connectors.
     const {
       t,
       showModal,
-      breakpoints: { isDesktop }
+      breakpoints: { isDesktop },
+      client
     } = this.props
+    const { hasConnector } = this.state
+
+    const appStoreUrl = generateUniversalLink({
+      cozyUrl: client.stackClient.uri,
+      slug: 'store',
+      subDomainType: client.getInstanceOptions().cozySubdomainType,
+      nativePath: `discover/?type=konnector&doctype=${DOCTYPE_CONTACTS}`
+    })
```

Comment:
We should not generate that in the render.

It should be done in a dedicated component (StoreButton or something)